models/user.model.js

```diff
@@ -0,0 +1,13 @@
+var mongoose = require('mongoose')
+var Schema = mongoose.Schema
+var passportLocalMongoose = require('passport-local-mongoose')
+
+var UserSchema = new Schema({
+    user_type: Number,
+    username: String,
+    password: String,
```

Comment:
Refer the [wiki](https://github.com/VITCMUN/munverse/wiki/Data-Models#user) for the right schema.